### PR TITLE
Problem sets already alphabetized. Updating TODO.

### PR DESCRIPTION
**Pull request by DavidMetcalfe**

### CODE REVIEW

**Comment by onebree** [comment]
When you can, please squash the commits into 1. If you need help or instruction, let me know!

File: javascript-frameworks-resources.md [hunk 1]
```diff
@@ -91,16 +91,16 @@
     * Lesson 15: Summary of the series (Coming soon...)
 
 ## Meteor
-* [Your First Meteor Application, A Complete Beginner’s Guide to the Meteor JavaScript Framework](http://meteortips.com/book/)
 * [BulletProof Meteor](https://bulletproofmeteor.com/)
+* [Your First Meteor Application, A Complete Beginner’s Guide to the Meteor JavaScript Framework](http://meteortips.com/book/)
 
 ## React
-* [React Tutorial](https://facebook.github.io/react/docs/tutorial.html)
```
Comment (vhf) [inline javascript-frameworks-resources.md]: I think you accidentally deleted this link.

File: javascript-frameworks-resources.md [hunk 2]
```diff
@@ -91,16 +91,16 @@
...
+* [Intro to the React Framework](http://code.tutsplus.com/tutorials/intro-to-the-react-framework--net-35660)
 * [Learning React.js: Getting Started and Concepts](https://scotch.io/tutorials/learning-react-getting-started-and-concepts)
-* [React.js Tutorial: Now is Your Time to Try It, Right in Your Browser](https://codegeekz.com/react-js-tutorial)
-* [React.js tutorial - Creating components](http://ghost.stevenisekimart.in/react-js-tutorial-creating-components)
+* [React Tutorial](https://facebook.github.io/react/docs/tutorial.html)
```
Comment (onebree) [inline javascript-frameworks-resources.md]: @vhf the deleted link [you mentioned](https://github.com/vhf/free-programming-books/pull/1671/files#r43247939) is right here, L100. I thought the same thing, too, at first

**Comment by vhf** [comment]
Thanks for your PR, looks almost good to me! :+1: 

@onebree I still prefer atomic commits for this project, as I said [here](https://github.com/vhf/free-programming-books/pull/1656#issuecomment-150563941) last week. :) (tl;dr: please don't squash if you can avoid it)

File: javascript-frameworks-resources.md (hunk 2 above)
Comment (vhf) [inline javascript-frameworks-resources.md]: Right, thank you! :+1:

**Comment by onebree** [comment]
@vhf I totally overlooked that comment, sorry. Just trying to help!

**Comment by vhf** [comment]
No problem! Your help is much appreciated!

**Pull request merged**